tests/unit/test-class-wc-payments.php

```diff
@@ -167,6 +167,40 @@ function () {
 		$this->assertNotEquals( 'woocommerce_rest_missing_nonce', $response->get_data()['code'] );
 	}
 
+	public function test_ajax_init_platform_checkout_sends_correct_data() {
```

Comment:
Since we're only asserting the Customer id in this test, should we name it something like "test_ajax_init_platform_checkout_creates_new_customer"?

Reply:
I also considered this but we may want to add additional data points that we would like to test, other than just customer ID (e.g. `user_id`, `email`, etc.). If so, we could add them here. 

Although, at the moment we _are_ only testing customer ID so `test_ajax_init_platform_checkout_sends_correct_customer_id` works just as well and we can always rename it in the future.

Thoughts?

Reply:
> Although, at the moment we are only testing customer ID so `test_ajax_init_platform_checkout_sends_correct_customer_id` works just as well and we can always rename it in the future.

I agree 👍. Let's keep the name consistent with its current state.

Reply:
This test only focus on the `customer_id` creation. I think it will involve additional logic to check on the other data.

Reply:
I don't think we need to check on other data right now, but renaming the test sounds good.

Reply:
done [d7e94e2](https://github.com/Automattic/woocommerce-payments/pull/4166/commits/d7e94e2f0346a8267d0d7a14c2f64ca202de0522)